content/applications/finance/payment_providers/razorpay.rst

```diff
@@ -23,6 +23,11 @@ Configuration on Razorpay Dashboard
    checkboxes are ticked.
 #. Click on :guilabel:`Create Webhook` to finalize the configuration.
 
+.. note::
+   The :guilabel:`Recurring payments` feature must be activated if you want to make recurring payments.
+   To get this feature, you need to send a request to the `Razorpay Support team
+   <https://razorpay.com/support/#request>`_.
```

Comment:
@auva-odoo Could this be simpler? Like "Send a request to the Razorpay support team to enable recurring payments."?